src/Providers/Anthropic/Enums/AnthropicCacheType.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace EchoLabs\Prism\Providers\Anthropic\Enums;
+
+enum AnthropicCacheType
+{
+    case ethemeral;
```

Comment:
I see value of keeping the enum here but Imo keeping provider related settings in the prisms code can make maintenance way harder. 

@sixlive can think different.

Reply:
Hilariously, having added the enum to avoid spelling mistakes, I misspelt it in the enum...! Now fixed.

Definitely not a huge loss if we don't include the enum. Really just a convenience thing.